### ISSUE TITLE
Dockerfile: missing skimage dependency

### ISSUE THREAD

**Issue by ajschadler12** [reuter-lab/qatools-python#33]
## Description
skimage is not installed in docker or singularity containers produced from the included Dockerfile.

## Steps to Reproduce

```shell
$ docker build -t latest -f Dockerfile .
# output truncated...

$ docker run --rm -u $(id -u):$(id -g) -v $SUBJECTS_DIR -v $out_dir $image_id --subjects-dir $SUBJECTS_DIR --output_dir $output_dir

-----------------------------
qatools-python
-----------------------------

Found subjects directory /home/ajschadler12/bids/derivatives/freesurfer
Found subject bert

ERROR: the 'skimage' package is required for running this script, please install.
```

## Expected Behavior

`scikit-image` is expected to be installed inside containers derived from the included `Dockerfile`

## Environment
 - qatools-python Version: b17c954
 - OS: Ubuntu 20.04 (modified docker file)
 - CPU: AMD Ryzen 7 5800H

## Additional Context

Adding `scikit-image` to requirements.txt might help ensure `scikit-image` is installed properly in Docker containers built from Dockerfile.

**Comment by kdiers** [comment]
Thank you very much for the report. We will provide a major update including changes to the dockerfile, requirements, and incorporation of new features from the dev branch. In the meantime, I'd suggest manually editing the requirements file or using the development version ([freesurfer-module-dev](https://github.com/Deep-MI/qatools-python/tree/freesurfer-module-dev)) if that works for you.

**Comment by annchenknodt** [comment]
Just wanted to say that i ran into the same thing, and also had to change six version numbers in the requirements.txt file since i kept running into errors like:
`Could not find a version that satisfies the requirement kiwisolver==1.3.2 (from versions: 0.1.0, 0.1.1, 0.1.2, 0.1.3, 1.0.0, 1.0.1, 1.1.0, 1.2.0, 1.3.0, 1.3.1)`
faced this both trying to run in python on my HPC (python v 3.6.8) and trying to build from the Dockerfile, but got it to work after bumping down those 6 version numbers and installing scikit-image.

Thanks for the tool!

**Comment by kdiers** [comment]
Thanks for the report, too! This is very helpful, and we'll address these issues in the next release, which is expected during June.

**Comment by kdiers** [comment]
Just a quick notice that we have released version 2.0 of this toolbox, which should solve these issues. If anything persists, please feel free to reopen this issue or write otherwise.
